Demo/src/main/res/layout/fragment_paypal.xml

```diff
@@ -21,6 +21,32 @@
 
     </com.google.android.material.textfield.TextInputLayout>
 
+    <com.google.android.material.textfield.TextInputLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+
+        android:hint="@string/paypal_phone_country_code">
+
+        <com.google.android.material.textfield.TextInputEditText
+            android:id="@+id/buyer_phone_country_code_edit_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content" />
```

Comment:
What if we set the input type to `phone`?

Reply:
Great suggestion! I just pushed the change.